testsupport/wait/awaitility.go

```diff
@@ -180,10 +175,10 @@ func (a *Awaitility) WaitForToolchainClusterWithCondition(t *testing.T, namespac
 
 // WaitForNamedToolchainClusterWithCondition waits until there is a ToolchainCluster with the given name
 // and with the given ClusterCondition (if it the condition is nil, then it skips this check)
-func (a *Awaitility) WaitForNamedToolchainClusterWithCondition(t *testing.T, name string, condition *toolchainv1alpha1.ToolchainClusterCondition) (toolchainv1alpha1.ToolchainCluster, error) {
-	t.Logf("waiting for ToolchainCluster '%s' in namespace '%s' to have condition '%v'", name, a.Namespace, condition)
+func (a *Awaitility) WaitForNamedToolchainClusterWithCondition(t *testing.T, name string, cdtype toolchainv1alpha1.ConditionType) (toolchainv1alpha1.ToolchainCluster, error) {
```

Comment:
not related to your changes in your PR, but this function doesn't seem to be used anywhere. If it's true, then we can easily remove it

Reply:
yes right, removed it now,
 i had thought to remove it in next refactor PR.